eng/pipelines/corefx-base.yml

```diff
@@ -173,6 +182,34 @@ jobs:
             - ${{ job.customBuildSteps }}
 
           - ${{ if eq(job.submitToHelix, 'true') }}:
+            - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
+              - script: $(_msbuildCommand) eng/publish.proj
+                      /t:PublishTestAssets
+                      /p:FilesToPublishPattern=$(Build.SourcesDirectory)/artifacts/helix/**/*.zip
+                      /p:AccountKey=$(dotnetfeed-storage-access-key-1)
+                      /p:ExpectedFeedUrl=$(_dotnetFeedUrl)
+                      /p:OSGroup=${{ parameters.targetOS }}
+                      /p:ArchGroup=$(_architecture)
+                      /p:ConfigurationGroup=$(_BuildConfig)
+                      /p:TargetGroup=$(_framework)
+                      /p:OfficialBuildId=$(Build.BuildNumber)
+                      /p:ContinuousIntegrationBuild=true
+                      /p:ManifestBuildId=$(Build.BuildNumber)
+                      /p:ManifestBuildData=Location=$(_dotnetFeedUrl)
+                      /p:ManifestBranch=$(Build.SourceBranchName)
+                      /p:ManifestCommit=$(Build.SourceVersion)
+                      /p:ManifestRepoUri=$(Build.Repository.Uri)
+                displayName: Publish test assets to dotnet-core feed
+                condition: and(succeeded(), eq(variables['_publishTests'], 'true'))
+
+              - task: PublishBuildArtifacts@1
+                displayName: Publish test asset manifest to artifacts container
+                inputs:
+                  pathToPublish: $(Build.SourcesDirectory)/artifacts/AssetManifests
+                  artifactName: manifests
```

Comment:
Could you named this: `BuildAssetsManifest` please? We're publishing another assets after we publish packages, so that all the .xml manifests are in the same container:

https://github.com/dotnet/corefx/blob/master/eng/pipelines/publish.yml#L73

Also, from you official build I noticed the .xml file name is not corefx-test-assets.xml and it will clash with the asset manifest that we produce later when publishing packages. Could you please make it be called corefx-test-assets.xml ?

You just need to pass AssetManifestName down to publish.proj:

https://github.com/dotnet/corefx/blob/master/eng/publish.proj#L11

Reply:
I don't think we need the publish artifacts step in corefx-base.yml because the one in publish.yml will publish them. 

I'm passing the corefx-test-assets.xml asset manifest name to publish.proj.

Reply:
Never mind about the removing the corefx-test-assets.xml publish step to the build artifacts.  It needs to be there and I need to include the build leg name in the artifactName. Like "Windows_NT_x64_Release" or "Linux_x64_Debug" to put it along side the build logs.

Reply:
> I don't think we need the publish artifacts step in corefx-base.yml because the one in publish.yml will publish them.

It will not publish them, because that step runs in a different machine, different environment, so the files will not be available in the publish step. We can talk about this offline if it helps.